### PR TITLE
Fix PHP version number for Magento 2.2.

### LINKED ISSUE #4118: Magento 2.2.x does not support PHP 7.2
# Bug report

## Description

DevDocs states that Magento 2.2.x supports PHP 7.2, which is not true.

Commit that made the change: https://github.com/magento/devdocs/commit/9c03365ae06e06eb23c4344b24f1018ff6952c3d#diff-2c0a624a75cf786e8905587c1042634d

Link to page in DevDocs: https://devdocs.magento.com/guides/v2.2/install-gde/system-requirements-tech.html

## Expected result

DevDocs should state that php 7.2 is not supported.

## Actual result

Upon running composer update (whilst running php 7.2) I receive:

```
$ bin/magento composer update
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - magento/product-community-edition 2.2.8 requires php ~7.0.13|~7.1.0 -> your PHP version (7.2.16) does not satisfy that requirement.
    - magento/product-community-edition 2.2.8 requires ext-mcrypt * -> the requested PHP extension mcrypt is missing from your system.
    - Installation request for magento/product-community-edition 2.2.8 -> satisfiable by magento/product-community-edition[2.2.8].
```

### PR DESCRIPTION
**Pull request by dobooth**
## This PR is a:

- Content fix or rewrite

## Summary

When this pull request is merged, it will...

Fix incorrect PHP version numbers for 2.2.

Fixes #4118

### CODE REVIEW

File: guides/v2.2/performance-best-practices/software.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ See [Magento 2.2.x technology stack requirements]({{ page.baseurl }}/install-gde
 
 Operating system configurations and optimizations are similar for Magento as other high-load web applications. As the number of concurrent connections handled by the server increases, the number of available sockets can become fully allocated. The Linux kernel supports a mechanism to "reuse" and "recycle" TCP connections. Be aware that more aggressive recycling than re-use may cause issues on the load balancers. To enable these kernel settings, set the following values in `/etc/sysctl.conf`: 
 
-```
+```bash
```
Comment (keharper) [inline guides/v2.2/performance-best-practices/software.md]: bash is for a single command line command. The CSS adds a "$ " before the first character on the first line only.

These are lines in a file, so the original three backticks was correct. Had this been a pair of commands, each line would need to be wrapped with the bash code gates individually.

Comment (dobooth) [reply guides/v2.2/performance-best-practices/software.md]: Ah thanks.

**Comment by bdenham** [comment]
I can only take the contributor's word for it in #4118. You'll need to confirm with someone in the know. Knowing which version of PHP works with various other versions of Magento core and extensions is a full-time job. :)

**Comment by dobooth** [comment]
I checked with the Pangolins and got this:
Magento v2.2.0 : https://github.com/magento/magento2ce/blob/2.2.0/composer.json#L11
Magento v2.2.1 : https://github.com/magento/magento2ce/blob/2.2.1/composer.json#L11
Magento v2.2.2 : https://github.com/magento/magento2ce/blob/2.2.2/composer.json#L11
Magento v2.2.3 : https://github.com/magento/magento2ce/blob/2.2.3/composer.json#L11
Magento v2.2.4 : https://github.com/magento/magento2ce/blob/2.2.4/composer.json#L11
Magento v2.2.5 : https://github.com/magento/magento2ce/blob/2.2.5/composer.json#L11
Magento v2.2.6 : https://github.com/magento/magento2ce/blob/2.2.6/composer.json#L11
Magento v2.2.7 : https://github.com/magento/magento2ce/blob/2.2.7/composer.json#L11
Magento v2.2.8 : https://github.com/magento/magento2ce/blob/2.2.8/composer.json#L11

From Magento 2.2.0 till Magento 2.2.4 : `"7.0.2|7.0.4|~7.0.6|~7.1.0"`
From Magento 2.2.5 till Magento 2.2.8 : `"~7.0.13|~7.1.0"`

File: guides/v2.2/performance-best-practices/software.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ The kernel parameter `net.core.somaxconn` controls the maximum number of open so
 
 ## PHP
 
-Magento fully supports 7.2.11. There are several factors to account for when configuring PHP to get maximum speed and efficiency on requests processing.
+Magento 2.2 fully supports up to version 7.1.0. There are several factors to account for when configuring PHP to get maximum speed and efficiency on requests processing.
```
Comment (keharper) [inline guides/v2.2/performance-best-practices/software.md]: @dobooth This wording doesn't sound right to me. Hold off on merging small_changes into master.

Comment (keharper) [reply guides/v2.2/performance-best-practices/software.md]: fully supports 7.0.13 and higher as well as 7.1.x.

File: _includes/install/php_2.2.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
-| 7.0.0&ndash;7.0.12 | 7.0.13&ndash;7.1.x | 7.2.0&ndash;7.2.11 |
-| ![Not supported]({{ site.baseurl }}/common/images/red-x.png) | ![Supported (2.1.2 and later)]({{ site.baseurl }}/common/images/green-check.png) | ![Supported (2.1.2 and later)]({{ site.baseurl }}/common/images/green-check.png) |
+| 7.0.0&ndash;7.0.12 | 7.0.13&ndash;7.1.0 | 7.2.0&ndash;7.2.11 |
```
Comment (keharper) [inline _includes/install/php_2.2.md]: This should probably be a four-column table.

Change the heading of the second column to 7.0.13 - 7.0.x (green check)
Insert a new column with the heading of 7.1.x (green-check)
Change the last column heading to 7.2.x (red-x)

The tildes mean greater than or equal to, without incrementing the values to the left of the last period.

Don't take my word as gospel. Get an architect or senior developer to review

**Pull request merged**